ocs_ci/ocs/ocp.py

```diff
@@ -1106,3 +1105,95 @@ def verify_cluster_operator_status(cluster_operator):
     log.info(f'{cluster_operator} status is valid')
 
     return True
+
+
+def validate_cluster_version_status():
+    """
+    Verify OCP upgrade is completed, by checking 'oc get clusterversion'
+    status
+
+    Returns:
+        bool: False in case that one of condition flags is invalid:
+        Progressing (should be False), Failing(should be False)
+        or Available (should be True)
+
+    """
+    ocp = OCP(kind="clusterversion")
+    operator_data = ocp.get('-o json', out_yaml_format=False)
+    conditions = operator_data['items'][0].get('status').get('conditions')
```

Comment:
what this `.get('conditions')` returns?  I guess list, can happen that this is different format like null, None or not define at all?  For more safe way I would go with  `.get('conditions', [])` in this case we will be OK all the time.

Reply:
done